release_operator_with_konflux.md

```diff
@@ -0,0 +1,112 @@
+```console
+releaseName=helm-operator-staging-$RANDOM
+oc create -f - <<EOF
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Release
+metadata:
+  name: $releaseName
+  namespace: orchestrator-releng-tenant
+spec:
+  releasePlan: helm-operator-staging
+  snapshot: $snapshot
+EOF
+```
+
+* Wait for the release to be validated:
+```console
+oc wait --for=condition=Validated=True release.appstudio.redhat.com/$releaseName
+```
+
+* Validate the outcome of the release by checking its processed status:
+```console
+status=$(oc get release.appstudio.redhat.com/$releaseName -ojsonpath='{.status.conditions[?(@.type=="Processed")].status}')
+statusMessage=$(oc get release.appstudio.redhat.com/$releaseName -ojsonpath='{.status.conditions[?(@.type=="Processed")].message}')
+if [ "$status" == "True" ]; then
+  echo Success: $statusMessage
+else
+  echo Failure: $statusMessage
+fi
+```
+
+* If the release failed, check the pipeline logs in the UI for further information.
+
+* Validate that the container images are in the staging registry by inspecting them with skopeo. The location of the images are defined in the RPA and the digest is the same as the one in the workspace we retrieved early to validate that the controller image pullspec and the one in the bundle matched.
+Run this command to extract the repository URL for the controller and the bundle for the staging environment (replace helm-operator-staging for helm-operator-prod for production):
+```console
+controllerRepository=$(oc get rpa -n rhtap-releng-tenant helm-operator-staging -ojsonpath='{.spec.data.mapping.components[?(@.name=="controller-rhel9-operator")].repository}')
+controllerDigest=$(oc get release.appstudio.redhat.com/$releaseName -ojsonpath='{.status.artifacts.images[?(@.name=="controller-rhel9-operator")].shasum}')
+bundleRepository=$(oc get rpa -n rhtap-releng-tenant helm-operator-staging -ojsonpath='{.spec.data.mapping.components[?(@.name=="orchestrator-operator-bundle")].repository}')
+bundleDigest=$(oc get release.appstudio.redhat.com/$releaseName -ojsonpath='{.status.artifacts.images[?(@.name=="orchestrator-operator-bundle")].shasum}')
+controllerPullSpecStaging=$controllerRepository"@"$controllerDigest
+bundlePullSpecStaging=$bundleRepository"@"$bundleDigest
+
+skopeo inspect docker://controllerPullSpecStaging >/dev/null && echo "Controller image found in $controllerPullSpecStaging" || echo "Controller image not found in $controllerPullSpecStaging"
+skopeo inspect docker://bundlePullSpecStaging >/dev/null && echo "Controller image found in $bundlePullSpecStaging" || echo "Controller image not found in $bundlePullSpecStaging"
+```
+
+
+* Update the FBC graph.yaml to include the new release. Run the `opm alpha render-basic` command and then update the resulting catalog manifest to ensure that it references the staging repository for the controller, since the build originating from the bundle's snapshot will point to the workspace location in quay.io.
```

Comment:
What's the location of `graph.yaml` ?

Reply:
Done a bit. But let me know what you think. In essence, the FBC repository should be self explanatory as it only contains a few files. Nevertheless it is perhaps clear to me but not to others that are not familiar with it, so don't hesitate to correct me 😄